website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -31,36 +28,43 @@ contract EvenNumber {
-[article-groth16]: https://www.risczero.com/news/on-chain-verification
-[article-proof-composition]: https://www.risczero.com/news/proof-composition
-[release-v0.10.0]: https://github.com/risc0/risc0-ethereum/releases/tag/v0.10.0
-[release-v0.9.0]: https://github.com/risc0/risc0-ethereum/releases/tag/v0.9.0
-[foundry-template]: https://github.com/risc0/bonsai-foundry-template
-[sepolia-verifier-v0.10.0]: https://sepolia.etherscan.io/address/0x4e9559f1fdf82dd1babdd42b4550ffdb1a1c59de#code
-[sepolia-verifier-v0.9.0]: https://sepolia.etherscan.io/address/0xe57255C10291771A2E14f7eb257555AE82D78302#code
+## Deployed verifiers
+
+All of our deployed verifier contracts implement the [IRiscZeroVerifier][IRiscZeroVerifier] interface.
+It is recommended that you use the [RiscZeroVerifierRouter][RiscZeroVerifierRouter], described below.
+
+### RiscZeroVerifierRouter
+
+Calls to `RiscZeroVerifierRouter.verify()` will be routed to the appropriate base verifier contract depending on which version of the zkVM was used to generate the receipt.
+By using the `RiscZeroVerifierRouter`, your contract can accept multiple types of receipts, including batch-verified receipts and receipts generated with future improvements to the zkVM and proofs system.
+
+RISC Zero deploys and manages the `RiscZeroVerifierRouter` contract listed below.
+This contract will have verifiers added for each release of the zkVM, and will have verifiers removed in the case of security vulnerabilities.
+You can find detailed information in the [version management design][VersionManagement], including information about how to manage your own copy of the smart contracts, if your application requires it.
+
+### Contract Addresses
+
+:::info
+
+Contracts are not deployed for unreleased versions.
+You can use the [deployed contracts for a released version][doc-released-contracts].
+
+:::
+
+<!-- TODO: Move this example into risc0-ethereum such that it will be under the same version management -->
+
+[EvenNumber.sol]: https://github.com/risc0/risc0-foundry-template/blob/main/contracts/EvenNumber.sol#L46-L52
 [Groth16Receipt]: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Groth16Receipt.html
+[IRiscZeroVerifier]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/IRiscZeroVerifier.sol
+[RiscZeroVerifierRouter]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/RiscZeroVerifierRouter.sol
+[VersionManagement]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/version-management-design.md
+[foundry-template]: https://github.com/risc0/risc0-foundry-template
 [term-image-id]: /terminology#image-id
 [term-journal]: /terminology#journal
 [term-receipt]: /terminology#receipt
 [term-verify]: /terminology#verify
 [term-zkvm-program]: /terminology#zkvm-program
-[VersionManagement@main]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/version-management-design.md
+[doc-released-contracts]: /api/blockchain-integration/contracts/verifier#contract-addresses
```

Comment:
This needs to be a relative path I think

Reply:
Basically, once we run the docusaurus version command to promote the next version, this URL will break. But if we keep it relative, it will work across promotions.

Reply:
The intention here is to link to the latest release, rather than the "next" release that the reader is currently viewing. So that's why it's absolute. Does that seem right to you? I can open up a follow-up PR to fix that. (I enabled auto-merge before I saw this)

Reply:
I wonder if we should be putting this kind of content that is 'versionless' under `docs` rather than the versioned `api`?